### ISSUE TITLE
Plugin does not show up in Obsidian after installation

### ISSUE THREAD

**Issue by workflowsguy** [aviskase/obsidian-link-indexer#13]
Hello,

i have installed version 0.0.7 of the plugin in Obsidian 0.9.17, but it does not show up anywhere in the GUI or the command palette.

**Comment by aviskase** [comment]
Thanks, you found the bug for clean installs :bug: 


Here is a workaround:

1. Close Obsidian
2. Go to you `<vault>/.obsidian/plugins/obsidian-link-indexer` directory
3. Open `data.json` file, and change its content to `{}` (instead of `undefined`)

After that plugin will be able to start.

I'll push the new version this weekend with #12 , it will bring the whole new system for settings and I'll make sure this bug won't be there :smile:

**Comment by chishaku** [comment]
Hi @aviskase I'm having the same issue.

I installed and enabled v1.0.0 and I'm on Obsidian 0.10.11.

I don't see any settings panel or command associated with this plugin.

I didn't see a `data.json` file in that path but added one anyways with an empty `{}` and still nothing.

Thanks in advance for any help.

**Comment by aviskase** [comment]
@chishaku hmm, just tested with new vault on Obsidian 0.10.11 and it's fine.

1. What's your operating system?
2. Do you have anything under `.obsidian/plugins/obsidian-link-indexer` ?

**Comment by meskill** [comment]
@aviskase 

Hello, I have the same issue as @chishaku - no settings panel or command on latest versions.

1. Windows 10
2. under `.obsidian/plugins/obsidian-link-indexer` there are two files: `main.js` and `manifest.json`

**Comment by aviskase** [comment]
@meskill  sorry for late reply. 
1) Open devtools by using key combination `Ctrl+Shift+I` . Open `Console` tab.
2) Go to Obsidian settings, `Community plugins` tab. Disable `Link indexer` plugin
3) Enable it again. Do you see any errors in the console?

**Comment by meskill** [comment]
I had `Link indexer` plugin disabled and after opening devtools and enabling it there were no errors and new tab in settings appeared.

But after reboot the obisidian it showed this in console:
```
Obsidian Developer Console
(unknown) Loading Folder Note plugin.
(unknown) Loading Changelog plugin
(unknown) Load Prettier Format plugin
(unknown) loading smart-random-note
(unknown) loading markdown-table-editor plugin
(unknown) Loading the Workbench plugin.
(unknown) Loading the Review plugin.
(unknown) Loading natural language date parser plugin
(unknown) loading plugin: Cycle through panes
(unknown) loading Find unlinked files and unresolved links plugin
index.html:23476 Uncaught (in promise) TypeError: Cannot read property 'view' of null
    at Object.eval [as checkCallback] (eval at evalFunc (app.js:1), <anonymous>:23476:32)
    at app.js:1
    at Array.filter (<anonymous>)
    at e.listCommands (app.js:1)
    at LinkIndexer.removeOwnCommands (eval at evalFunc (app.js:1), <anonymous>:2322:27)
    at LinkIndexer.reloadSettings (eval at evalFunc (app.js:1), <anonymous>:2303:14)
    at LinkIndexer.eval (eval at evalFunc (app.js:1), <anonymous>:2282:30)
    at step (eval at evalFunc (app.js:1), <anonymous>:71:23)
    at Object.eval [as next] (eval at evalFunc (app.js:1), <anonymous>:52:53)
    at fulfilled (eval at evalFunc (app.js:1), <anonymous>:42:58)
app.js:1 Uncaught TypeError: Cannot read property 'settings' of undefined
    at AdvancedAppearance.refreshSystemTheme (eval at evalFunc (app.js:1), <anonymous>:281:23)
    at t.e.tryTrigger (app.js:1)
    at t.e.trigger (app.js:1)
    at t.trigger (app.js:1)
    at t.<anonymous> (app.js:1)
    at app.js:1
    at Object.next (app.js:1)
    at s (app.js:1)
```

After some tries it seems like it working for me only if I start obisidian with disabled `Link Indexer` and then enable it

**Comment by aviskase** [comment]
@meskill I tried reproducing on windows box, and I couldn't =( I have one idea left: when you go to "About" screen, there should be two lines: current version and installer version. What is the installer version?

On Linux I once stumbled upon a problem when installer version was very old and even though Obsidian code itself was updated, the surrounding electron environment was not. It caused some plugins to fail. To fix it I downloaded a new installer and reran it.
